bear-language-requirements.yaml

```diff
@@ -0,0 +1,355 @@
+ApertiumLintBear: !!set
```

Comment:
we do not need these to be sets.

simple lists would be ok.

the set is coming from `bear.LANGUAGES`.  You should convert that to a list and sort it.

Reply:
I made the requested changes @jayvdb

Reply:
This is not addressed.